db/import.go

```diff
@@ -549,6 +563,9 @@ func (db *DB) Import(in *json.Decoder, vault *vault.Client) error {
 				if err != nil {
 					return err
 				}
+			case "errors":
+				err := db.importErrors(in)
+				return err
```

Comment:
You can just return the call, since you're not doing anything with the temporary variable.